README.md

```diff
@@ -1,14 +1,27 @@
-# Yaapii.Atoms
+<a href="https://icarus-consult.de">
+    <img src="https://xpertify.de/wp-content/uploads/cache/images/icarus-consulting-gmbh-lueneburg-logo/icarus-consulting-gmbh-lueneburg-logo-850777872.png" alt="Icarus logo" title="Icarus" align="right" width="200" />
```

Comment:
think we should remove our logo here, since it is Open Source.

But you can add the EO badge: http://www.elegantobjects.org/badge.svg